capgains/transactions_reader.py

```diff
@@ -23,19 +23,29 @@ class TransactionsReader:
     ]
 
     @classmethod
-    def get_transactions(cls, csv_file):
+    def get_transactions(cls, csv_file, reverse=False):
         """Convert the CSV-file entries into a list of Transactions."""
         transactions = []
         try:
             with open(csv_file, newline='') as f:
                 reader = csv.reader(f)
                 last_date = None
+                if reverse:
+                    reader = list(reader)
+                    reader.reverse()
                 for entry_no, entry in enumerate(reader):
+                    # Skip lines that start with #, they are comments.
+                    # Skip empty lines
+                    if (len(entry) > 0 and entry[0].startswith("#")) or \
+                       (len(entry) == 0):
+                        continue
                     actual_num_columns = len(entry)
                     expected_num_columns = len(cls.columns)
                     if actual_num_columns != expected_num_columns:
                         # Each line in the CSV file should have the same number
                         # of columns as we expect
+                        if reverse:
+                            entry_no = len(reader)-entry_no
```

Comment:
We should provide spaces around the binary operator `-`. I think flake8 will point this out since it's part of the PEP8 style guide.